nntrainer/include/tensor.h

```diff
@@ -157,14 +157,22 @@ class Tensor {
 
   /**
    * @brief     Substract Tensor Element by Element
-   * @param[in] m Tensor to be added
+   * @param[in] m Tensor to be subtracted
    * @retval    Calculated Tensor
    */
   Tensor subtract(Tensor const &m) const;
 
+  /**
+   * @brief     memcpyless version of subtract
+   * @param[in] value value to be subtracted
+   * @retval #ML_ERROR_NONE  Successful
+   * @retval #ML_ERROR_INVALID_PARAMETER Invalid Parameter
+   */
+  int subtract_i(float const &value);
+
   /**
    * @brief     subtract value Element by Element
-   * @param[in] value value to be added
+   * @param[in] value value to be subtracted
```

Comment:
IMO, value to subtract? e.g., subtract value from tensor.

Reply:
Thanks. Updated :)